### PR TITLE
Cluster V1: added feature gates & admission controllers at kubernetes options

### PR DESCRIPTION
**Pull request by pavelpereyma**
Added feature gates and admission controllers to cluster kubernetes options.

### CODE REVIEW

File: pkg/v1/cluster/schemas.go [hunk 1]
```diff
@@ -150,9 +150,13 @@ func (result *View) UnmarshalJSON(b []byte) error {
 }
 
 // KubernetesOptions represents additional k8s options such as pod security policy,
-// feature gates and etc.
+// feature gates (Alpha stage only) and admission controllers.
 type KubernetesOptions struct {
 	// EnablePodSecurityPolicy indicates if PodSecurityPolicy admission controller
 	// must be turned on/off.
 	EnablePodSecurityPolicy bool `json:"enable_pod_security_policy"`
+	// FeatureGates represents feature gates that should be enabled.
```
Comment (kolsean) [inline pkg/v1/cluster/schemas.go:158]: Please add a blank line before the comment similar to the code style used in the package.
This remark applies for this and the following comment.

File: pkg/v1/cluster/testing/fixtures.go [hunk 2]
```diff
@@ -32,7 +32,15 @@ const testGetClusterResponseRaw = `
         "enable_patch_version_auto_upgrade": true,
         "zonal": false,
         "kubernetes_options": {
-            "enable_pod_security_policy": true
+            "enable_pod_security_policy": true,
+			"feature_gates": [
```
Comment (kolsean) [inline pkg/v1/cluster/testing/fixtures.go:36]: Let's use spaces for indentation here and in other similar pieces below. Tab characters break format of these JSON fixtures a little. You can view it more clearly using `git log -1 -p` command.

File: pkg/v1/cluster/testing/fixtures.go [hunk 3]
```diff
@@ -927,8 +1081,14 @@ const testCreateClusterResponseRaw = `
...
         "kubernetes_options": {
-            "enable_pod_security_policy": false
-        }
+            "enable_pod_security_policy": false,
+			"feature_gates": [
+        		"CSIMigrationOpenStack"
+      		],
+			"admission_controllers": [
+				"LimitRanger"
+			]
```
Comment (kolsean) [inline pkg/v1/cluster/testing/fixtures.go:1090]: Extra spaces  after `]` character here and below.

File: pkg/v1/cluster/testing/fixtures.go [hunk 4]
```diff
@@ -1294,8 +1456,14 @@ const testUpdateClusterWithEnabledPSPOptsRaw = `
...
         "kubernetes_options": {
-            "enable_pod_security_policy": true
-        }
+            "enable_pod_security_policy": true,
+			"feature_gates": [
+        		"CSIMigrationOpenStack"
+      		],
+			"admission_controllers": [
+				"LimitRanger"
+			]
```
Comment (kolsean) [inline pkg/v1/cluster/testing/fixtures.go:1465]: and here.

File: pkg/v1/cluster/testing/fixtures.go [hunk 5]
```diff
@@ -1332,8 +1504,13 @@ const testUpdateClusterWithEnabledPSPResponseRaw = `
...
         "kubernetes_options": {
-            "enable_pod_security_policy": true
-        }
+            "enable_pod_security_policy": true,
+			"feature_gates": [
+        		"CSIMigrationOpenStack"
+      		],
+			"admission_controllers": [
+				"LimitRanger"
+			]        }
```
Comment (kolsean) [inline pkg/v1/cluster/testing/fixtures.go:1513]: Apart from the extra tabs I guess the line break is missing after `]`

**Review by kolsean — CHANGES_REQUESTED** [review]
Looks good, but there are some formatting notes.

**Comment by pavelpereyma** [comment]
Thanks for review. I updated formatting for docs and some code style.

**Review by kolsean — APPROVED** [review]
LGTM

**Pull request merged**